pkg/cmd/roachtest/tests/libpq_blocklist.go

```diff
@@ -19,7 +19,36 @@ var libPQBlocklists = blocklistsForVersion{
 	{"v23.1", "libPQBlocklist23_1", libPQBlocklist23_1, "libPQIgnorelist23_1", libPQIgnorelist23_1},
 }
 
-var libPQBlocklist23_1 = libPQBlocklist22_2
+var libPQBlocklist23_1 = blocklist{
```

Comment:
can you change the test so it only has one blocklist, similar to how django_blocklist was changed? we don't need per-version maps, since we can now just have one list that is different on each release branch

Reply:
Done. Will change the other versions' in a bit.